js/utils.js

```diff
@@ -1596,6 +1596,26 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     });
   }
 
+  function toFormattedString(options) {
+    if (!options) {
+      return '';
+    }
+
+    if (Array.isArray(options)) {
+      options = options.filter(function(item) {
+        return !!item.trim();
```

Comment:
If there's an `undefined` in the array, this will trigger an error.

![image](https://user-images.githubusercontent.com/290733/106595693-d5de4b80-654b-11eb-9488-332fb58df99f.png)

Reply:
@tonytlwu, fixed